setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
Could you please add a space between the equal sign here on both sides? In the setup.py file, I more or less follow PEP 8, so spaces are added around "=" when it is not an argument keyword assignment?